### PR TITLE
DDCNLS-5811: Update libraries for Play 2.5.19

### PR DESCRIPTION
**Pull request by daniel-manning**
*Update libraries for Play 2.5.19
*Move to build.sbt

### CODE REVIEW

File: app/controllers/EasinessController.scala [hunk 1]
```diff
@@ -42,9 +44,9 @@ class EasinessController @Inject()(
                                         formProvider: EasinessFormProvider
                                       ) extends FrontendController with I18nSupport with Enumerable.Implicits {
```
Comment (matthew-mcnally) [inline app/controllers/EasinessController.scala]: Is "Enumerable.Implicits" the correct way of extending the class?

File: build.sbt [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    pipelineStages in Assets := Seq(concat,uglify),
+    // only compress files generated by concat
+    includeFilter in uglify := GlobFilter("helplinefeedbackfrontend-*.js")
+  )
+
+lazy val testSettings: Seq[Def.Setting[_]] = Seq(
+  fork := true
+)
+  
+
```
Comment (matthew-mcnally) [inline build.sbt]: Double space

File: app/controllers/EasinessController.scala [hunk 3]
```diff
@@ -17,19 +17,21 @@
...
+import models.Easiness
 import pages.EasinessPage
+import play.api.mvc.{Action, AnyContent}
 import utils.{Enumerable, Navigator}
 import views.html.easiness
 
 import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
 
 class EasinessController @Inject()(
```
Comment (matthew-mcnally) [inline app/controllers/EasinessController.scala]: Remove this linebreak

File: app/controllers/EasinessController.scala (hunk 1 above)
Comment (daniel-manning) [reply app/controllers/EasinessController.scala]: Presumably puts it in the correct scope ahead of other implicits.

File: app/controllers/EasinessController.scala (hunk 3 above)
Comment (daniel-manning) [reply app/controllers/EasinessController.scala]: what?

**Pull request merged**